pkg/api/router/webhook.go

```diff
@@ -26,5 +26,13 @@ import (
 func MountWebhooks(r *gin.Engine, co *apisix.ClusterOptions) {
 	// init the schema client, it will be used to query schema of objects.
 	_, _ = validation.GetSchemaClient(co)
-	r.POST("/validation/apisixroutes/plugin", gin.WrapH(validation.NewPluginValidatorHandler()))
```

Comment:
Is the plugin validation missing?

Reply:
Nope, it's in the validation of ApisixRoute now.